src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -1089,4 +1089,7 @@ Do you want to proceed?</value>
   <data name="Inline_Diagnostics_Warning" xml:space="preserve">
     <value>Inline Diagnostics - Warning</value>
   </data>
+  <data name="Execute_Suggested_Action" xml:space="preserve">
+    <value>Execute Suggested Action</value>
```

Comment:
Not that i could find.  This string itself is taken from teh editor (it's what they use).  Since we TakeOwnership and provide our own TWD, i wanted to preserve the string.